01/app.js

```diff
@@ -0,0 +1,25 @@
+const form = document.querySelector('form');
+
+
+if(form) {
+  form.addEventListener('submit', handleSubmit)
+}
+
+
+function handleSubmit(e) {
+  e.preventDefault()
+
+  const firstName = e.target.elements.firstName.value.trim();
+  const lastName = e.target.elements.lastName.value.trim()
+
+  if(!firstName || !lastName) return
```

Comment:
W przypadku imienia i nazwiska warto raczej sprawdzić ich długość - można wstawić jedną literkę i nie sądzę, aby były to poprawne dane ;) - tak na przyszłość